src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -110,7 +110,7 @@ def test_source_waveform_parallel_broadcast(multi_instrument_session):
         timeout=5)
 
     pass_fail = multi_instrument_session.get_site_pass_fail(site_list='')
-    assert pass_fail == [True, True]
+    assert pass_fail == {0: True, 1: True}
```

Comment:
can you change this to use non-contiguous sites? To verify keys are ok.

Reply:
Added a new test to verify non-contiguous sites.